### PR TITLE
ADD flexusd.eth Space

### PR DESCRIPTION
**Pull request by c01nfl3x**
Change to add flexusd.eth space

### CODE REVIEW

File: spaces/aliases.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "unfederalreserve": "unfederalreserve.eth",
   "yam": "yam.eth",
   "velotoken": "velotoken.eth",
-  "cream": "cream-finance.eth"
+  "cream": "cream-finance.eth",
+  "flexUSD": "flexusd.eth"
 }
```
Comment (gokaiorg) [inline spaces/aliases.json:8]: You need to edit this file only if you migrate an existing space to a space created with ENS. For a new space, just follow this documentation https://docs.snapshot.page/guides/create-a-space

**Review by gokaiorg — CHANGES_REQUESTED** [review]

File: spaces/aliases.json (hunk 1 above)
Comment (c01nfl3x) [reply spaces/aliases.json:8]: Thanks @gokaiorg . That was done already but when I go to created space page it shows a blank screen.

Comment (gokaiorg) [reply spaces/aliases.json:8]: Do you have access to this page and if so, did you fill in the required fields? https://snapshot.page/#/flexusd.eth/settings

Comment (c01nfl3x) [reply spaces/aliases.json:8]: Yes, I left out without filling only Members and Filters, the rest is filled.

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Your space is displayed but it's not a space migration, use your PR to add your logos or your sub-domain but you have to delete the aliases .json files.

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.
If you want to add avatars please follow this link https://docs.snapshot.page/guides/add-avatars

**Pull request closed without merging**